### ISSUE TITLE
[Accessibility]: Line chart should have custom arrow keyboard navigation 

### ISSUE THREAD

**Issue by vovakulikov** [sourcegraph/sourcegraph#44046]
### Audit type

Keyboard navigation

### Problem description

At the moment, Line Chart (and actually any other charts like bar or pie charts) does use simple link sequence  (or circle element with tabIndex="0") So data points are focusable but in case of many points chart, you have to go through all points even if you simply want to reach the last data point on the chart. 

### Expected behavior

We should implement an arrow-like navigation system over data points on the line chart. So it would be possible to navigate to different data points with →↑←↓ keys. 

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

/cc @joelkw @felixfbecker @vovakulikov

**Comment by vovakulikov** [comment]
It seems like a really nice to have, but I wouldn't consider this as P0 since it doesn't block us in any way for the external a11y audit. P1 then

**Comment by chwarwick** [comment]
p1 if it's required by audit
